docs/creating-studies.md

```diff
@@ -80,8 +80,13 @@ Most users have a workflow that looks like this:
   - Build your study with the CLI to make sure your queries load correctly.
 
 We use [sqlfluff](https://github.com/sqlfluff/sqlfluff) to help maintain a consistent
-style across many different SQL query authors. There are two commands you may want to
-run inside your study's directory:
+style across many different SQL query authors. You don't have to if you are working
+on development, but if you're going to submit your study when completed back to
+the Cumulus project, it may be worth mimicing our linting strategy. You can copy our 
+[sqlfluff configuration rules](https://github.com/smart-on-fhir/cumulus-library-core/blob/main/pyproject.toml)
+into your study directory to configure the right behavior.
```

Comment:
so - i :could: automatically create a .sqlfluff file in study directories? I'd have to move the config out of the pyproject.toml though, which i'm softly reluctant to do. We also might expect that we just have to do this on our end during a PR.

Reply:
I'm fine with that as a maintainer burder. Only risk is that we'd be making SQL changes to an already proven pile of SQL, and folks may be pretty reluctant at that point.